src/plugins/voice-cooperation/components/VoiceFloatingNew/VoiceFloatingNew.tsx

```diff
@@ -0,0 +1,428 @@
+                    <span className={styles.username}>
+                      {members.filter((member) => member.isSpeaking)[0].userInfo.name}
+                      {members.filter((member) => member.isSpeaking)[0].clientId ===
+                        voicePlugin.teamworkManager.userInfo.clientId && <span>&nbsp;(You)</span>}
+                    </span>
+                    <span className={styles.tip}>{voicePlugin.msg("plugins.voiceCooperation.speakingOnly")}</span>
+                  </div>
+                )}
+                {members.filter((member) => member.isSpeaking).length > 1 && (
+                  <div className={styles.speakingTipsContainer}>
+                    <span className={styles.tip}>{voicePlugin.msg("plugins.voiceCooperation.speakingOnly")}</span>
+                  </div>
+                )}
+              </div>
+            </div>
+          )}
+          <div className={styles.buttonGroup}>
+            <Button
+              className={styles.controlButton}
+              sx={{
+                width: 24,
+                height: 24,
+                p: 0,
+              }}
+              onClick={onToggleMicrophone}
+            >
+              <span
+                style={{
+                  width: 24,
+                  height: 24,
+                }}
+              >
+                {isMicrophoneMuted ? (
+                  <MicrophoneSlashIcon color="red" />
+                ) : (
+                  <MicrophoneIcon
+                    sx={{
+                      color: "var(--theme-color-g300)",
+                    }}
+                  />
+                )}
+              </span>
+            </Button>
+            <Button
+              className={styles.controlButton}
+              sx={{
+                width: 24,
+                height: 24,
+                p: 0,
+              }}
+              onClick={handleToggle}
+            >
+              <span
+                style={{
+                  width: 10,
+                  height: 10,
+                }}
+              >
+                {isExpanded ? <UparrowIcon /> : <DownarrowIcon />}
+              </span>
+            </Button>
+          </div>
+        </Box>
+        {isExpanded && (
+          <div>
+            <div className={styles.divider}></div>
+            <Box id="voice-floating">
+              <div
+                style={{
+                  overflow: "overlay",
+                  maxHeight: 236,
+                }}
+                className={styles.memberContainer}
+              >
+                {members.map((member) => {
+                  return (
+                    <MemberListItem key={member.userInfo.id} {...member} onToggleMicrophone={onToggleMicrophone} />
+                  );
+                })}
+              </div>
+            </Box>
+            <div className={styles.divider}></div>
+            <Button className={styles.leaveButton} variant="ghost" onClick={onLeave}>
```

Comment:
Internationalization has not been implemented

Reply:
1